botCommands/simpleCommands.js

```diff
@@ -43,7 +43,31 @@ registerBotCommand(question.regex, question.cb);
 
 const data = {
   regex: /(?<!\S)\/data(?!\S)/,
-  cb: () => '**Please state your question in the form of a question! https://www.dontasktoask.com/**',
+  cb: () => {
+    const dataEmbeed = new Discord.MessageEmbed()
```

Comment:
I'd love if you fixed the typo "dataEmbeed", should be "dataEmbed".

Reply:
Done